src/Components/Web/src/Virtualization/Virtualize.cs

```diff
@@ -66,6 +66,12 @@ public sealed class Virtualize<TItem> : ComponentBase, IVirtualizeJsCallbacks, I
     [Parameter]
     public RenderFragment<TItem>? ItemContent { get; set; }
 
+    /// <summary>
+    /// Gets or sets the empty content.
+    /// </summary>
+    [Parameter]
+    public RenderFragment? EmptyContent { get; set; }
```

Comment:
Could you follow the instructions here: https://github.com/dotnet/aspnetcore/blob/main/docs/APIBaselines.md#publicapiunshippedtxt?

Reply:
Yes sir. I missed that part of the instructions.